apps/README.md

```diff
@@ -0,0 +1,26 @@
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
+
+## Launch `FluentUI Tester` app on iOS
+
+Prereq: FluentUI Tester on iOS can only run on a Mac.
+
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire Fluent UI React Native repository. E.g. from the root of the repo:
+```
+    yarn && yarn build
+```
+
+2. Then go into `apps/ios/src` folder and run pod install:
+
+```
+    cd apps/ios/src
+    pod install
+```
+Note: if you get the error: "CocoaPods could not find compatible versions for pod "MicrosoftFluentUI"," you will need to run `pod repo update` and then run `pod install` again.
+
+3. Return to the ios directory and run yarn ios to launch the FluentUI Tester app:
+
+```
+    cd ..
+    yarn ios
+```
+Note: the first time you yarn ios, you receive an error and have to run "FluentUITester.xcworkspace" directly from Xcode. The workspace can be found in the apps/ios/src folder. After running the workspace the first time from Xcode, you will be able to `yarn ios` from the CLI.
```

Comment:
Troubleshooting:
- If you have the packager didn't launch in a separate terminal, you can run `yarn start` from `apps/ios` to launch it separately
- If you want to do direct debugging via xcode, after the `pod install`, you can launch `src/FluentUITester.xcworkspace` and build/run the scheme "ReactTestApp"
- If you want to have a clean rebuild of the generated iOS project, you can do the following:
```
cd apps/ios/
rm src/FluentUITester.xcworkspace
rm -r src/Pods/
pod install --project-directory=src.
```

Reply:
Re troubleshoooting- `git clean -xfd` would be a sure-fire way to wipe away all differences between your local repository and the one you cloned from upstream

Reply:
That command can also remove uncommited changes you have so it's also dangerous :D 
I dunno that I would want to mention it in the documentation for setting up the test app without caveats (could cause more troubles while troubleshooting)

Reply:
Hah yes it can come with a warning on what it does. It’s also all documented in git

Reply:
Added changes in new commit. Let me know if you have any additional feedback, otherwise feel free to resolve comment.